web/views.py

```
@@ -5286,6 +5286,16 @@ def upload_educational_video(request):
     return render(request, "videos/upload.html", {"form": form})
 
 
+def play_youtube_videos(request, video_title, video_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `play_youtube_videos`

(ANN201)